cmake/tpl/caliper.cmake

```diff
@@ -1 +1,3 @@
-set(${lib_name}_libs libcaliper.a)
+if(ENABLE_TIMER)
+  set(${lib_name}_libs libcaliper.a)
+endif()
```

Comment:
We should move this logic to cmake/InstallTPLs.cmake so as to not even look for a caliper package